### ISSUE TITLE
"Property 'build' does not exist on  XMLBuilder" - Typescript ts(2339)

### ISSUE THREAD

**Issue by mia-z** [NaturalIntelligence/fast-xml-parser#397]
* [x] Are you running the latest version?
* [ ] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [ ] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)?

### Description
When using XMLBuilder, Typescript doesn't see the Builder object as having the build() function. It works a intended, however. I assume this is just a Typescript problem.
### Input
- Import XMLBuilder
- initialize a new instance
- try using the build() function
- typescript will complain that build() doesnt exist on XMLBuilder

### Code
```js
import { XMLParser, XMLBuilder } from "fast-xml-parser";
...

const builder = new XMLBuilder({ ignoreAttributes: true })
const xmlAsJson = {
  a: true
};
const builtXml = builder.build(xmlAsJson);
                         ^^^^^
```

### Output
The output works as expected

### expected data
Typescript should not complain about the missing function

**Would you like to work on this issue?**

- [ ] Yes
- [x] No

[Bookmark](https://github.com/NaturalIntelligence/fast-xml-parser/stargazers) this repository for further updates.

Extra things to note:
Using Visual Studio Code via WSL
Using Yarn 3.1.0 with pnp enabled
Using Yarn's VSCode Typescript SDK

**Comment by amitguptagwl** [comment]
Looks like you're on old version. This issue was already fixed. Please confirm

**Comment by mia-z** [comment]
Using fast-xml-parser 4.0.0-beta.2
Still saying "build" doesn't exist on XMLBuilder.

Updated to Typescript 4.5.2 and Yarn 3,1,1, problem persists.

**Comment by amitguptagwl** [comment]
My bad, I have not published the fix. Let me do that. I'll update

**Comment by amitguptagwl** [comment]
Please check with v4.0.0-beta.3
